packages/frontend/src/pages/ParkingSpace/components/ParkingSpaceInfo.tsx

```diff
@@ -115,7 +127,29 @@ export const ParkingSpaceInfo = (props: { listingId: number }) => {
             </Box>
           </Box>
         </Box>
-        <Box border="1px solid black" flex="1" />
+        <Box
+          flex="1"
+          sx={{
+            width: '100%',
+            height: '100%',
+            cursor: 'pointer',
+          }}
+          onClick={() =>
+            window.open(
+              getMapImageUrl(parkingSpaceListing.rentalObjectCode),
+              '_blank'
+            )
+          }
+        >
+          <Box
+            component="img"
+            src={getMapImageUrl(parkingSpaceListing.rentalObjectCode)}
+            alt="parking space map image"
+            sx={{
+              height: '100%',
+            }}
+          />
+        </Box>
```

Comment:
Wouldn't it be easier to wrap the img in a <a href="..."/> ?

Reply:
What is the link intended to go to? 

From what I read it's the recommended way to use it in a Box with the component prop = "img". But if there's another way I'm happy to fix that :)

Reply:
Ah, I meant something like this
```
    <a href={getMapImageUrl(parkingSpaceListing.rentalObjectCode)} target="_blank">
      <Box
        component="img"
        src={getMapImageUrl(parkingSpaceListing.rentalObjectCode)}
        alt="parking space map image"
        sx={{
          height: '100%',
        }}
      />
    </a>
```
But yeah its no biggie